app/controllers/movie_actors_controller.rb

```diff
@@ -0,0 +1,17 @@
+class MovieActorsController < ApplicationController
+  def create
+    movie_actor = MovieActor.new(create_movie_actor_params)
+
+    unless movie_actor.save
+      flash[:alert] = "Error: #{movie_actor.errors.full_messages}"
```

Comment:
Great use of this controller & setup for this story - it's exactly how I like to approach it as well!